homeassistant/components/tesla_fleet/manifest.json

```
@@ -8,5 +8,5 @@
   "iot_class": "cloud_polling",
   "loggers": ["tesla-fleet-api"],
   "quality_scale": "gold",
-  "requirements": ["tesla-fleet-api==0.7.3"]
+  "requirements": ["tesla-fleet-api==0.7.4"]
```

```suggestion
  "requirements": ["tesla-fleet-api==0.7.8"]
```
Might have to do this in a seperate PR so that it also bumps Tessie and Teslemetry

---

I was able to update Tesla Fleet to 0.7.8 but not Tessie and Teslemetry.